### ISSUE TITLE
how to get the label file of HMDB51

### ISSUE THREAD

**Issue by wen-flow** [craston/MARS#28]
I sincerely appreciate the provided code.I want to know how to get the label file of HMDB51. I only downloaded the video file.
Thank you very much.

**Comment by devendraswamy** [comment]
same issue , have you solved this issue ?

**Comment by wen-flow** [comment]
yes! you can give an  E-Mail Address here and I will send the label files as soon as posibile.

**Comment by interfaceFeng** [comment]
I have same problem, so I want to get the label file of HMDB51, UCF101 and kinetics. My email is 1391512991@qq.com.
Thank very much.

**Comment by interfaceFeng** [comment]
> yes! you can give an E-Mail Address here and I will send the label files as soon as posibile.

I have same problem, so I want to get the label file of HMDB51, UCF101 and kinetics. My email is 1391512991@qq.com.
Thank very much.

**Comment by praveenr2998** [comment]
I too need the labels file, E-mail - praveenr2998@gmail.com.
Thank you very much.

**Comment by baoqianyue** [comment]
I also need the labels file, My E-mail - baoqianyue@163.com.
Thank you very much.

**Comment by interfaceFeng** [comment]
you can get if from its official website, there are 3 splits in&nbsp;HMDB51



---Original---
From: "Barack ***@***.***&gt;
Date: Thu, Jul 8, 2021 18:21 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [craston/MARS] how to get the label file of HMDB51 (#28)




 
I also need the labels file, My E-mail - ***@***.***
 Thank you very much.

**Comment by baoqianyue** [comment]
> you can get if from its official website, there are 3 splits in&nbsp;HMDB51
> […](#)
> ---Original--- From: "Barack ***@***.***&gt; Date: Thu, Jul 8, 2021 18:21 PM To: ***@***.***&gt;; Cc: ***@***.******@***.***&gt;; Subject: Re: [craston/MARS] how to get the label file of HMDB51 (#28) I also need the labels file, My E-mail - ***@***.*** Thank you very much. — You are receiving this because you commented. Reply to this email directly, view it on GitHub, or unsubscribe.

Thanks!

**Comment by huwenfeng233** [comment]
I too need the labels file, E-mail - [huwenfeng2@gmail.com](mailto:huwenfeng2@gmail.com).
Thank you very much.
